format_test.go

```diff
@@ -9,6 +9,8 @@ import (
 	qt "github.com/frankban/quicktest"
 )
 
+type myBytes []byte
```

Comment:
Maybe move this close to nilStringer after the test?

Reply:
done